src/blueprint/from_config/operators/call.rs

```diff
@@ -0,0 +1,127 @@
+use std::collections::hash_map::Iter;
+use std::collections::BTreeMap;
+
+use crate::blueprint::*;
+use crate::config;
+use crate::config::{Config, Field, GraphQLOperationType, KeyValues};
+use crate::lambda::{Expression, IO};
+use crate::mustache::{Mustache, Segment};
+use crate::try_fold::TryFold;
+use crate::valid::Valid;
+
+fn find_value<'a>(args: &'a Iter<'a, String, String>, key: &'a String) -> Option<&'a String> {
+  args
+    .clone()
+    .find_map(|(k, value)| if k == key { Some(value) } else { None })
+}
+
+pub fn update_call(
+  operation_type: &GraphQLOperationType,
+) -> TryFold<'_, (&Config, &Field, &config::Type, &str), FieldDefinition, String> {
+  TryFold::<(&Config, &Field, &config::Type, &str), FieldDefinition, String>::new(
+    move |(config, field, _type_of, name), b_field| {
+      let Some(call) = &field.call else {
+        return Valid::succeed(b_field);
+      };
+
+      if validate_field_has_resolver(name, field, &config.types).is_succeed() {
+        return Valid::fail(format!(
+          "@call directive is not allowed on field {} because it already has a resolver",
+          name
+        ));
+      }
```

Comment:
Should implement `resolvable_directives` in `config.rs`

Reply:
done